### DISCUSSION TITLE
Remapping ; to :

### DISCUSSION THREAD

**Discussion by mfigurski80** [helix-editor/helix#4064] [Q&A] [answered]
Hey all!

Sorry if this is a feature somewhere already, but I've been struggling to enable this in helix. I'm coming from vim, where in my vimrc I added (to adjust for my bored/clumsy typing style):

```
" Issues typing :W instead of :w... this allows me to not touch shift key
:nnoremap ; :
```

This allows me to touch the ; key like shift is already pressed, and then I can add whatever other command I want. I'm aware this might be some kind of programming sin -- however, vim is not nice to an accidental `:W`. Is there a way to enable something like this in helix? This is like the only thing tripping up my workflow in helix rn, and I'd love to slowly start porting my things to the platform. I've tried adding something like the following:

```
[keys.normal]
";" = ":"
```

It seems this fails because ':' is not a name of a typeable command, because everything has to be a complete command  it seems. Is there a way to map to a partial command?

**Comment by kirawi** [comment, marked as answer]
I think it's: `";" = "command_mode"`

**Reply by mfigurski80** [reply]
Thank youuu
:w
